app/main/view.ts

```diff
@@ -45,6 +45,34 @@ export class View extends BrowserView {
 				this.webContents.setAudioMuted(true);
 			});
 		}
+
+		this.webContents.addListener('did-navigate-in-page', () => {
+			const isSettingsPage = this.url.includes('renderer/preference.html');
+			if (isSettingsPage) {
+				return;
+			}
+			this.canGoBackButton();
```

Comment:
Is this function call for showing the go back button if we can, or to be more specific enable the button. If this is the case I wonder if `maybeEnableGoBackButton` would be a better name.

Reply:
I have used the same name as in webview.ts. Should i change this ?

Reply:
Sure, `maybeEnableGoBackButton` sounds better.

Reply:
Ok i'll change the name here.

Reply:
`maybeEnableGoBackButton` sounds like a hackish function name to me.